docs/topics/class-based-views/generic-display.txt

```diff
@@ -334,7 +341,8 @@ Here, we have a URLconf with a single captured group::
 
     # urls.py
     from django.urls import path
-    from books.views import PublisherBookListView
+
+    from .views import PublisherBookListView
```

Comment:
Leave alone. This looks like a root URLConf since it uses "books/<publisher>/".

Reply:
ok